client/src/search.js

```diff
@@ -27,7 +27,9 @@ export class SearchWidget extends React.Component {
     redirectTo: null,
     searchResults: [],
     serverError: null,
-    showSearchResults: true
+    showSearchResults: true,
+    locale:
+      (this.props.pathname && this.props.pathname.split("/")[1]) || "en-US"
```

Comment:
Please don't copy things into state from props. Keep it in props. 
If it gets weird to type `this.props.pathname.split("/")[1] || "en-US"` all over the place, write a method on the component called something like `getCurrentLocale()`